### PR TITLE
facebook auth

### PR DESCRIPTION
**Pull request by HoangCacLoanChau**
## Jira link
https://capstone-alumni.atlassian.net/jira/software/c/projects/AL/boards/1?modal=detail&selectedIssue=AL-21
## Describe your changes
add facebook provider
add env facebook client ID and secret
## Screenshot/Record (FE)
image
image

## Postman (BE)
image
image

## Checklist before requesting a review
1. [x] I have performed a self-review of my code
2. [x] Have you lint your code locally before submission?

### CODE REVIEW

**Comment by HoangCacLoanChau** [comment]
I have not handled this case: can not sign in by google with the same email which signs up by Facebook.

**Comment by HoangCacLoanChau** [comment]
> Please add CLIENT_ID and CLIENT_SECRET to file .env.example

Ok, done

**Comment by pvnamGit** [comment]
Also create task on Jira for your issue nha

**Comment by HoangCacLoanChau** [comment]
> can not sign in by google with the same email which signs up by Facebook.

ok, done nha

File: src/pages/api/auth/[...nextauth].ts [hunk 1]
```diff
@@ -45,7 +46,12 @@ export const nextAuthOptions = {
         }
       },
     }),
+    FacebookProvider({
+      clientId: process.env.FACEBOOK_CLIENT_ID || '',
+      clientSecret: process.env.FACEBOOK_CLIENT_SECRET || '',
+    })
   ],
+
```
Comment (congtrung6391) [inline src/pages/api/auth/[...nextauth].ts:54]: no new line

File: src/pages/api/auth/[...nextauth].ts [hunk 2]
```diff
@@ -45,7 +46,12 @@ export const nextAuthOptions = {
         }
       },
     }),
+    FacebookProvider({
```
Comment (congtrung6391) [inline src/pages/api/auth/[...nextauth].ts:49]: re-order provder: Google -> Facebook -> Credentials

**Review by congtrung6391 — COMMENTED** [review]
Please add facebook signIn action to `SignUpForm`

**Review by pvnamGit — APPROVED** [review]
LGTM

**Pull request merged**